packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -2,7 +2,9 @@
 	position: relative;
 	display: flex;
 	flex-flow: column;
-	overflow: hidden;
+	&.is-isolated {
```

Comment:
Should we add a comment prior to this rule explaining why `overflow: hidden` is only applied when `.is-isolated` is applied? It's a tricky edge case this one, so could be good to make sure we don't accidentally reintroduce the problem further down the track.